xarray/core/common.py

```diff
@@ -593,17 +593,12 @@ def pipe(
             temperature_c  (lat, lon) float64 16.53 13.85 21.27 18.37
             precipitation  (lat, lon) float64 2.731 2.719 2.848 2.467
 
-        >>> x.pipe(adder, arg=2).pipe(div, arg=2)
-        <xarray.Dataset>
-        Dimensions:        (lat: 2, lon: 2)
-        Coordinates:
-        * lon            (lon) int64 150 160
-        * lat            (lat) int64 10 20
-        Data variables:
-            temperature_c  (lat, lon) float64 8.264 6.923 10.63 9.185
-            precipitation  (lat, lon) float64 1.366 1.359 1.424 1.234
-
-        >>> x.pipe(adder, arg=2).pipe(div, arg=2).pipe(sub_mult, sub_arg=2, mult_arg=2)
+        >>> (
+        ... x
+        ... .pipe(adder, arg=2)
+        ... .pipe(div, arg=2)
+        ... .pipe(sub_mult, sub_arg=2, mult_arg=2)
+        ... )
```

Comment:
I had written this in a rush—would you mind replacing with this? Then the formatting looks normal again. Apologies for the back & forth

```
(
    x
    .pipe(adder, arg=2)
    .pipe(adder, arg=4)
    .pipe(div, arg=2)
    .pipe(sub_mult, sub_arg=2, mult_arg=2)
)
```